webapp/components/admin_console/mfa_settings.jsx

```diff
@@ -86,7 +86,7 @@ export default class MfaSettings extends AdminSettings {
                     helpText={
                         <FormattedHTMLMessage
```

Comment:
We're supposed to be moving away from using FormattedHTMLMessage to using FormattedMessage with nested placeholders for the links instead.

Deferring to someone else on whether that should block this PR, or whether it can be dealt with later.